src/Microsoft.Azure.WebJobs.Host.Storage/LegacyConfig.cs

```diff
@@ -1,35 +1,32 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Options;
 using Microsoft.WindowsAzure.Storage;
-using System;
-using System.Collections.Generic;
-using System.Text;
 
 namespace Microsoft.Azure.WebJobs
 {
 
     // $$$ Validate these?  And what are their capabilities? 
     public class LegacyConfigSetup : IConfigureOptions<LegacyConfig>
```

Comment:
I'm not sure why this LegacyConfigSetup/LegacyConfig stuff was added recently as new public surface area. We shouldn't have this